rfcs/0001-incentivization-transactions.md

```diff
@@ -0,0 +1,66 @@
+# Summary
+
+[summary]: #summary
+
+Provide a mechanism to pay users for having a verified identity and continuously adding data to their data host.
+
+# Motivation
+
+[motivation]: #motivation
+
+The protocol is deeply dependant on the network effects of users wanting applications and applications wanting users.
+To grow faster, we want to directly incentivize users to be on the network and create data.
+
+We expect users to proactively set up their identity, data capture, and data hosting to receive FCL tokens.
+
+# Guide explanation
+
+[guide-explanation]: #guide-explanation
+
+You can receive a portion of FCL minting by:
+
+1. Hosting a signed Fractal ID message.
+1. Regularly publishing the Merkle root of all data in your data host.
+
+# Reference explanation
+
+[reference-explanation]: #reference-explanation
+
+Mint tokens for addresses that publish the following:
+
+1. A unique ID signed by Fractal ID.
+1. A Merkle root containing more data than the previously published one for that Fractal ID.
+
+We will mint tokens with the following constraints:
+
+1. Constant amount per user per day, when below total issuance per day.
+   This ensures users are indifferent towards more users using the system.
```

Comment:
I'm not sure I agree with this, because while it helps with not discouraging late adopters (by flooring their upside), it hurts encouraging early ones (by capping it). Additionally, rewarding early adopters outsizedly seem to be in line with the expectations of the community and the practices of other projects.

Reply:
My concern is that if we issue a constant amount daily divided among users, then users are incentivized to limit the number of others getting these rewards. As it is, we can tune the max amount to be quite high, but not an entire day's worth of minting. We can also tune the amount so that it starts being evenly distributed at N total users, whatever we decide a good N is.

Reply:
That's a valid concern. Setting a max makes sense to me